internals/garbage-collector.rst

```diff
@@ -1,14 +1,17 @@
+.. _garbage-collector:
 .. _gc:
+.. _garbage_collector:
 
-Design of CPython's Garbage Collector
-=====================================
+==========================
+ Garbage Collector Design
+==========================
 
 :Author: Pablo Galindo Salgado
```

Comment:
Shouldn't this be lowercase or is it case-insensitive?

Reply:
> this be lowercase or is it case-insensitive?

Which bit are you referring to? The title or the author field?

A

Reply:
I didn't change it in this PR, so I'll mark as resolved.

A